setup.py

```diff
@@ -25,6 +25,7 @@
       version='0.3.dev0',
       description='cliquet',
       long_description=README,
+      license='APLv2.0',
       classifiers=[
           "Programming Language :: Python",
           "Topic :: Internet :: WWW/HTTP",
```

Comment:
This is version 1 I believe, and I don't see the version 2 [in the list of trove classifiers](https://pypi.python.org/pypi?%3Aaction=list_classifiers). Maybe we should ask them to add one and in the meantime just add a "v2" at the end.